domain/sp/service.go

```diff
@@ -87,6 +108,21 @@ func (svc Service) enrichWithContactInfo(sp *domain.ServiceProvider) error {
 	return nil
 }
 
+func (svc Service) enrichWithEndpoint(sp *domain.ServiceProvider) error {
+	if sp.Id == "" {
+		return nil
+	}
+	didDocument, _, err := svc.VDRClient.Get(sp.Id)
+	if err != nil {
+		return domain.UnwrapAPIError(err)
+	}
+	_, endpointURL, err := didDocument.ResolveEndpointURL(domain.NutsCommService)
+	if err == nil {
+		sp.Endpoint = endpointURL
+	}
```

Comment:
Just to make sure. The nuts comm endpoint of the customer should be a reference to the nuts comm endpoint of the node right? Is this actually the case or will this resolve the actual endpoint?

Reply:
> Just to make sure. The nuts comm endpoint of the customer should be a reference to the nuts comm endpoint of the node right? 

That's right.

> Is this actually the case or will this resolve the actual endpoint?

This is the ServiceProvider service (a.k.a. vendor), so this resolve should return the service endpoint as-is. Which it actually does (contrary to the name of the `ResolveEndpointURL` function): it does not resolve references.